docs/generating-code-from-schema.md

```diff
@@ -58,7 +59,33 @@ Please ensure that your project’s sources refer to the generated code using<!-
 </div> 
 
 You can exclude parts of the schema from code-generation by placing them in a different schema directory that is not specified<!-- http://go/pv --> as part of the `schemaPaths` for the plugin.
- 
+
+### Fixing the "Could not initialize class graphql.parser.antlr.GraphqlLexer" problem
+
+Gradle's plugin system uses a flat classpath for all plugins, which makes it very easy to run into classpath conflicts.
+One of the dependencies of the Codegen plugin is ANTLR, which is unfortuanatly used by some other plugins as well.
+If you see an error such as `Could not initialize class graphql.parser.antlr.GraphqlLexer` this typically indicates a classpath conflict.
+If this happens, please change the ordering of the plugins in your build script.
+ANTLR is typically backwards, but not forwards, compatible.
+
+For multi-module projects means you need to declare the Codegen plugin in the root build file, without applying it:
+
+```groovy
+plugins {
+    id("com.netflix.dgs.codegen") version "[REPLACE_WITH_CODEGEN_PLUGIN_VERSION]" apply false
+    
+    //other plugins
+}
+```
+
+In the module where the plugin should be applied, you specify the plugin in the plugins block again, but without the version.
+
+```groovy
+plugins {
+    id("com.netflix.dgs.codegen")
+}
+```
```

Comment:
Should we also add one for when using `buildscript` dependencies with `apply` plugin?

Reply:
added